core/server/levels.js

```diff
@@ -211,18 +211,19 @@ Meteor.methods({
 
     return createLevel({ templateId });
   },
-  updateLevel(name, position, hide) {
+  updateLevel(name, position, hide, featuresPermissions) {
     if (!this.userId) throw new Meteor.Error('missing-user', 'A valid user is required');
     check(name, String);
     check(position, { x: Number, y: Number });
     check(hide, Boolean);
+    check(featuresPermissions, Object);
```

Comment:
We probably want to do something like this, at least:

```suggestion
    check(featuresPermissions, { shout: String, globalChat: String, punch: String });
```

Ideally we'd want to check against the possible dropdown values.